rules/windows/process_creation/proc_creation_win_susp_lolbin_non_c_drive.yml

```diff
@@ -0,0 +1,29 @@
+title: Wscript Execution from Non C Drive
+id: 5b80cf53-3a46-4adc-960b-05ec19348d74
+status: experimental
+description: Detects Wscript or Cscript executing from a drive other than C. This has been observed with Qakbot executing from within a mounted ISO file.
+author: Aaron Herman
+date: 2022/10/01
+references:
+    - https://github.com/pr0xylife/Qakbot/blob/main/Qakbot_BB_30.09.2022.txt
+tags:
+    - attack.execution
+    - attack.t1059
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_lolbin:
+        CommandLine|contains:
+            - 'wscript.exe'
+            - 'cscript.exe'
```

Comment:
```suggestion
        Image|endswith:
            - '\wscript.exe'
            - '\cscript.exe'
```

Reply:
I added the leading `\` but it needs to use `contains` because in the Any.Run you'll see that there are quotes around both `(w|c)script.exe` and what's being called.

Reply:
What Any.run shows and what's in the log are two different things.

Reply:
You're right - updated!